### PR TITLE
lcio: changes to install and CPATH for python bindings

### PR DESCRIPTION
**Pull request by andresailer**
As far as I can tell, these are the minimal changes to the spack recipe to allow one to run 
```
$ python -c "import pylcio"
Loading LCIO ROOT dictionaries ...
```
with a spack installed LCIO. See also https://github.com/iLCSoft/LCIO/issues/106 

@vvolkl I am open for any tweaks to this of course.

### CODE REVIEW

**Review by adamjstewart — APPROVED** [review]
Just waiting for a review from @gaede @vvolkl

**Comment by adamjstewart** [comment]
Flake8:
```
var/spack/repos/builtin/packages/lcio/package.py:84: [E501] line too long (82 > 79 characters)
var/spack/repos/builtin/packages/lcio/package.py:85: [E501] line too long (94 > 79 characters)
var/spack/repos/builtin/packages/lcio/package.py:86: [E501] line too long (93 > 79 characters)
```

**Review by vvolkl — APPROVED** [review]
Looks good to me - thanks @andresailer !

**Review by gaede — APPROVED** [review]

**Comment by andresailer** [comment]
I fixed the style warnings from flake8

**Pull request merged**
